pkg/controller/networkpolicy/validate.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2019 Antrea Authors
+//
```

Comment:
should we keep the http handler code under pkg/apiserver/handlers as same as endpointQuerier?